Make Robot Work Better

boinary opened the pull request:
I made drivetrain work

Commit:
Oh my gosh!

I fixed the manual owo

KenwoodFox commented on src/main/java/frc/robot/subsystems/ExampleSubsystem.java:
```diff
@@ -7,14 +7,23 @@
+  public static void Manual (TalonSRX starboard, TalonSRX port, Joystick driverStick)
+  {
+    
+    Double thro = driverStick.getRawAxis(1);
```
Good work, variables look nice and have appropriate names. Try adding in code for a joystick, you can use the code in joe-drivetrain as an example.
The axis for yaw is 2.

KenwoodFox commented on src/main/java/frc/robot/subsystems/ExampleSubsystem.java:
```diff
@@ -7,14 +7,23 @@
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 /**
  * An example subsystem.  You can replace me with your own Subsystem.
```
Add some docs in about the subsystem!

KenwoodFox commented on src/main/java/frc/robot/subsystems/ExampleSubsystem.java:
```diff
@@ -7,14 +7,23 @@
 /**
  * An example subsystem.  You can replace me with your own Subsystem.
  */
 public class ExampleSubsystem extends Subsystem {
```
I highly recommend renaming your subsystem.

KenwoodFox reviewed (changes requested):
Looks great, fixup the name of the subsystem, dont forget that the file name also needs to be changed.
Try out adding in yaw, remember that the example code is in `joe-drivetrain`

Commit:
Added Yaw

added some yaw stuff

KenwoodFox committed:
Returned Example subsystem

Confirmed to compile properly.

Co-Authored-By: DirDraggo <boinary@users.noreply.github.com>

KenwoodFox reviewed (approved):
Changes approved, DO NOT MERGE until tested on the robot!

KenwoodFox commented on src/main/java/frc/robot/subsystems/ManualDrive.java:
```diff
@@ -0,0 +1,37 @@
+/**
+ * An example subsystem.  You can replace me with your own Subsystem.
+ */
+public class ManualDrive extends Subsystem {
```
`ManualDrive` doesn't really well define the subsystem, I would recomend calling it `Drivetrain` or `DriveBase`, you may put methods in here that dont well define with manual drive

KenwoodFox reviewed (changes requested):
I recommend making this change before we merge this into master.

Commit:
Added Robot.java, RobotMap.java

init variables in RobotMap.jave
Robot.java makes it work!

KenwoodFox commented on src/main/java/frc/robot/Robot.java:
```diff
@@ -50,7 +51,9 @@ public void robotInit() {
-  public void robotPeriodic() {
+  public void robotPeriodic() 
+  {
+    Manual();
```
This line is incorrect, i have no idea what its supposed to do, i recommend removing it

KenwoodFox commented on src/main/java/frc/robot/Robot.java:
```diff
@@ -120,6 +123,7 @@ public void teleopInit() {
   @Override
   public void teleopPeriodic() {
     Scheduler.getInstance().run();
+    DriveTrain.Manual(RobotMap.starboardMotor, RobotMap.portMotor, RobotMap.driverStick);
```
Your robot map is totally empty, so none of these variables mean anything.

KenwoodFox replied:
`DriveTrain.Manual();` Does not exist, I think you meant to use `ManualDrive.Manual();`

KenwoodFox reviewed (changes requested):
Make these changes asap.

Commit:
Fixed Build Errors

Deleted unneccessary code, added robotmap from master. Joe gave me a thumbs up and a wink.

Commit:
Fixed Yaw

Yaw Yeet. Added yaw to thro instead of subtracting.

Commit:
pushback

Commit:
changed TalonSRX

removed TalonSRX and added it with the good library

Commit:
Added SmartBoard Features

its not a bug, its a feature!

Commit:
Added to SmartDashboard

if you are looking for some smartdashboard features with thro and yaw, yaw in luck!

Commit:
Made new command OperatorControl

created OperatorControl command branch

moved ManualDrive.Manual() to Operator Control command

KenwoodFox reviewed (approved):
Looks good dude! 🥇

Commit:
Changed SmartDashboard

changed thro and yaw values to display when going forward, backward, not moving. Turning right, left, and not turning.

Commit:
is code working?

i dunno, why dont u check the smartdashboard?

Commit:
revised with Ethan

java gives me weird erros but still compiles. It's concerning

Commit:
made smartdashboard more specific

changed not moving output to "NOT MOVING"
